src/components/table/table-row/table-row.style.js

```diff
@@ -0,0 +1,238 @@
+import { css } from 'styled-components';
```

Comment:
I'm finding this file a little on the long side, could we split it?

Reply:
Good point, I've split it into `modern` and `classic`